src/runtime_src/core/common/ishim.h

```diff
@@ -203,6 +203,10 @@ struct ishim
   virtual xclGraphHandle
   open_graph(const xrt::uuid&, const char*, xrt::graph::access_mode am) = 0;
 
+  virtual xclGraphHandle
+  open_graph(const xrt::hw_context&, const char*, xrt::graph::access_mode)
+  { throw not_supported_error{__func__}; }
+
```

Comment:
Remove

```suggestion
```

Reply:
Hi @stsoe , 
We need this as Shim should handle open the graphs based on hw_context now.
This PR doesn't have shim changes to handle hardware context, Those changes would come in next PRs. Am i missing anything?